common-lib/src/main/scala/com/gu/mediaservice/model/ExternalThrallMessage.scala

```diff
@@ -47,7 +47,7 @@ object ExternalThrallMessage{
 
   implicit val createMigrationIndexMessage = Json.format[CreateMigrationIndexMessage]
 
-  implicit val writes = Json.writes[ExternalThrallMessage]
+  val writes = Json.writes[ExternalThrallMessage] // intentionally not implicit, so individual formats will be used
```

Comment:
what's the intention behind this?

Reply:
there's an ambiguous implicit resolution when trying to publish say `CreateMigrationIndexMessage` where the compiler can't decide between `writes` and `createMigrationIndexMessage` further up in the `object ExternalThrallMessage`  (the compiler error doesn't actually affect this PR tbf but best to get it right up front).

Reply:
is this right though? this gives me a bit of a feeling it's covering up a different implicit mistake (that i made)

Reply:
hmm I think it makes sense, and the overall writes is used explicitly further up the file in the `toJson` which is in-turn used in the logMarkers. let me take another look though, might be something else I can do to improve

Reply:
yep found an alternative, which leaves it as an implicit